### PR TITLE
Add Documentation for Adding Tools

### PR DESCRIPTION
**Pull request by Emuentes**

### CODE REVIEW

File: docs/drafts/onboarding-tools [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+1. Propose the tool here in the Package Maintenance Team repo by adding an issue describing your idea
+
+2. Put your proposal on the [agenda for the next meeting](https://github.com/nodejs/package-maintenance/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+meeting+)
+
+3. Discuss with the group during our team meeting so we can plan next steps. (See [Calendar](https://calendar.google.com/calendar/embed?src=nodejs.org_nr77ama8p7d7f9ajrpnu506c98@group.calendar.google.com) for date/time)
+
+4. Build the tool as a public GitHub repo that follows Package Maintenance Team standards
+- https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/Baseline%20practice%20-%20Document%20support%20levels.md
+
+5. Release your tool as a public NPM module (following our [documented steps for publishing modules](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/PUBLISH-GUIDELINES.md))
```
Comment (ljharb) [inline docs/drafts/onboarding-tools]:
```suggestion
5. Release your tool as a public npm module (following our [documented steps for publishing modules](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/PUBLISH-GUIDELINES.md))
```

File: docs/drafts/onboarding-tools [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+6. **As Needed:** Follow the steps for [transferring you repo into the org.](https://github.com/nodejs/admin/blob/0f2af76492b9efff8bafedbf650bb0a3c4f76e44/transfer-repo-into-the-org.md)
```
Comment (ljharb) [inline docs/drafts/onboarding-tools]:
```suggestion
6. **As Needed:** Follow the steps for [transferring your repo into the org](https://github.com/nodejs/admin/blob/0f2af76492b9efff8bafedbf650bb0a3c4f76e44/transfer-repo-into-the-org.md).
```

Comment (Eomm) [inline docs/drafts/onboarding-tools]: I would use the master link https://github.com/nodejs/admin/blob/master/transfer-repo-into-the-org.md

Comment (Emuentes) [inline docs/drafts/onboarding-tools]:
```suggestion
6. **As Needed:** Follow the steps for [transferring you repo into the org.](https://github.com/nodejs/admin/blob/master/transfer-repo-into-the-org.md)
```

Comment (Emuentes) [reply docs/drafts/onboarding-tools]: updated @Eomm

**Comment by Emuentes** [comment]
Changes implemented @Eomm @ljharb

Thanks for the review.

I'll leave this issue open until we discuss in the meeting on the 13th.

**Pull request merged**